main.js

```diff
@@ -1,54 +1,83 @@
-// var list = [];
+/* eslint-disable */
```

Comment:
Please do not disable linters altogether. If you're having issues having multiple classes in the same file, just disable that rule but do not disable ESLint in its entirety.

Reply:
Changed the same.